tenable/sc/organization.py

```diff
@@ -1,6 +1,26 @@
 # -*- coding: utf-8 -*-
-"""
-"""
+'''
+organization
+============
+
+The following methods allow for interaction with the Tenable.sc
+`Organization <https://docs.tenable.com/sccv/api/Organization.html>`_ API. These
+
+items are typically seen under the **Organization** section of Tenable.sc. 
+
+Methods available on ``sc.organization``:
+
+.. rst-class:: hide-signature
+.. autoclass:: OrganizationAPI
+
+    .. automethod:: _constructor
```

Comment:
its a minor thing, but the constructor is a private method, and should not be included in the docs.